.gitignore

```diff
@@ -1,3 +1,4 @@
+/.github/
```

Comment:
The `.github/` folder is used for issue templates + GitHub Actions workflows. It cannot be ignored!